Source/DafnyPipeline.Test/DafnyPipeline.Test.csproj

```diff
@@ -0,0 +1,23 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net5.0</TargetFramework>
+
+    <IsPackable>false</IsPackable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="DiffPlex" Version="1.7.0" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.8.3" />
+    <PackageReference Include="xunit" Version="2.4.1" />
+    <PackageReference Include="xunit.runner.visualstudio" Version="2.4.3">
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
```

Comment:
Interesting, does this allow you to actually fully package the tests and run them independently of the source tree? I don't know if that's particularly useful mind you. :)

Reply:
I don't think so. Documentation indicates the default for `IncludeAssets` is `all`, so this would remove some assets from the build. I got this project file by using `dotnet new xunit`. One of the missing options is `compile`, which means the types from this reference are not available for importing in this solution. Sounds reasonable.